Source/QuartzCore additions/CGPathAdditions.m

```diff
@@ -21,7 +21,7 @@ CGFloat fixInfinity(CGFloat inputFloat){
     return inputFloat;
 }
 
-CGPoint *fixPointsInfinity(CGPathElement *element){
+CGPoint *fixPointsInfinity(const CGPathElement *element){
```

Comment:
I see what you mean.

I think the correct fix is to change line 98 from:

void applyPathTranslation (void *info, const CGPathElement *element) {

to:

void applyPathTranslation (void *info, CGPathElement *element) {

?